geoblacklight/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
This doesn't seem like a good idea. It's almost certainly not going to work how one might assume it will work. Better integrate it into the willow repo-wide .gitignore at the root of our repository.